Makefile

```diff
@@ -141,6 +142,11 @@ build-container: BIN_DIR:=$(BIN_DIR)/$(GOOS)
 build-container: build ## Builds provisioner container image locally
 	$(CONTAINER_RUNTIME) build -f Dockerfile -t $(IMAGE) $(BIN_DIR)
 
+kind: 
```

Comment:
Testing locally, I believe that you can remove this entirely. The target you defined on 223 should handle this and you just need to call `$(KIND)`. You can also remove line 10 as it isn't needed either. 

Maybe there is some environmental skew that is making it so I'm not experiencing the need for this?